<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>authform</class>
 <widget class="QWidget" name="authform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,30,10,0,0">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="auth_label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Авторизация&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="reg_label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Регистрация&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="auth_horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="auth_lables_verticalLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="log_label">
          <property name="text">
           <string>Логин</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pas_label">
          <property name="text">
           <string>Пароль</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="email_label">
          <property name="text">
           <string>Почта</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="surname_label">
          <property name="text">
           <string>Фамилия</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="name_label">
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="check_label">
          <property name="text">
           <string>student/teacher</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="lines_verticalLayout" stretch="0,0,0,0,0,0">
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>60</number>
        </property>
        <item>
         <widget class="QLineEdit" name="log_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="pas_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="email_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="surname_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="name_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="check_lineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttons_horizontalLayout">
      <item>
       <widget class="QPushButton" name="sign_in_pushButton">
        <property name="text">
         <string>Вход</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sign_up2_pushButton">
        <property name="text">
         <string>Вход</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sign_up_pushButton">
        <property name="text">
         <string>Регистрация</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exit_pushButton">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="error_label">
      <property name="text">
       <string>Ошибка авторизации</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="error_reg_label">
      <property name="text">
       <string>Пользователь с таким именем уже существует</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
